enterprise/frontend/src/embedding-sdk/lib/polyfill-use-sync-external-store.ts

```diff
@@ -1,11 +1,13 @@
 import React from "react";
 import { useSyncExternalStore } from "use-sync-external-store/shim";
 
-import { isReact17OrEarlier } from "metabase/lib/react-compat";
```

Comment:
Uhm I'm not sure about this change in this file, I think for the polyfill, we only care if `useSyncExternalStore` is in React or not, we're using the version as a proxy, but doing so we could replace a polyfill they already put in place